### DISCUSSION TITLE
Config Map Support for helm charts when using argocd

### DISCUSSION THREAD

**Discussion by pratyushmck** [argoproj/argo-cd#8150] [General]
Hello Team, We had a scenario where we were looking forward to your opinion- We utilize an enterprise-wide helm chart that has extensive vetted logic. The chart is provided with a values,yaml file and additionally with config map and secrets files via the helm --set-file argument. These are application specific and housed in an external git repository. Our query is when we use argocd coupled with helm for deploying our applications, how do we handle this use case via the declarative setup in argocd?

**Comment by jgwest** [comment]
Hi @pratyushmck , I'm by no means a Helm expert, but does this help inform your use case: https://github.com/argoproj/argocd-example-apps/tree/master/helm-dependency ? This is an example of using Argo CD to consume a helm chart containing child  chart dependencies. You would then specify the Git repo/path to a folder containing these files within your Argo CD `Application`.
